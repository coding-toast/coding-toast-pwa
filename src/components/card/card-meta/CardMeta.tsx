import * as React from 'react'

interface CardMetaProps {
  publishDate: string
}

const CardMeta: React.FC<CardMetaProps> = props => {
  const { publishDate } = props

  return <p>{publishDate}</p>
}

export default CardMeta
