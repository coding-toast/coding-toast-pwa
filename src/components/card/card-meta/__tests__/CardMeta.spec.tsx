import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import CardMeta from '../CardMeta'

describe('<CardMeta />', () => {
  it('renders correctly', () => {
    const component = shallow(<CardMeta publishDate="November 11, 2016" />)
    expect(toJson(component)).toMatchSnapshot()
  })
})
