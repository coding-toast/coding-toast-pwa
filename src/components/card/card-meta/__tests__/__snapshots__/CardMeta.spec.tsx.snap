// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardMeta /> renders correctly 1`] = `
<p>
  November 11, 2016
</p>
`;
